src/net.c

```diff
@@ -651,7 +653,7 @@ int net__tls_load_verify(struct mosquitto__listener *listener)
 }
 
 
-#ifndef WIN32
+#if !defined(WIN32) && !defined(_AIX)
```

Comment:
This means you should probably update https://github.com/eclipse/mosquitto/blob/master/man/mosquitto.conf.5.xml#L1096 to include AIX there too.

Reply:
Sure.